pennylane/pulse/transmon.py

```diff
@@ -72,7 +72,7 @@ def transmon_interaction(
     see e.g. `arXiv:1804.04073 <https://arxiv.org/abs/1804.04073>`_,
     `arXiv:2203.06818 <https://arxiv.org/abs/2203.06818>`_, or `arXiv:2210.15812 <https://arxiv.org/abs/2210.15812>`_.
 
-    .. note:: Currently only supporting ``d=2`` with qudit support planned in the future.
+    .. note:: Currently only supporting ``d=2`` with qudit support planned in the future. For ``d=2``, we have :math:`a:=\frac{1}{2}(\sigma^x + i \sigma^y)`.
```

Comment:
I know you're just copying the other docstring, but why is this `sigma^x` instead of `sigma_x`? In other places in the docs we write `sigma_x` for the Pauli X matrix.

Reply:
Oh I didn't know the sub/superscripts had this much discrepancy across the docs. Sure, you've convinced me. Maybe we can just stick the tidying up on the backlog then.